styles/components/_section-title.scss

```diff
@@ -0,0 +1,106 @@
+@import "../global/index";
+
+.usda-section__container {
+  margin-top: rem(15);
+  margin-bottom: rem(15);
+  flex-wrap: wrap;
+  .usda-section-title__container {
+    flex-wrap: wrap;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    @include for-phone-only {
+      flex-wrap: nowrap;
+    }
+    .usda-section-title__title-icon {
+      margin-left: rem(15);
+      align-self: flex-end;
+      @include for-phone-only {
+        margin-left: rem(5);
+        align-self: center;
+      }
+    }
+    .usda-section-title__header {
+      display: flex;
+      flex-direction: column;
+      margin-left: rem(9);
+      margin-right: auto;
+      align-self: flex-end;
+      flex-wrap: wrap;
+      strong.usda-section-title__overline {
+        font-size: rem(12);
+        flex-wrap: wrap;
+        font-weight: $font-semibold;
+        text-transform: uppercase;
+        color: #555;
+      }
+      .usda-section-title__title {
+        display: flex;
+        align-items: flex-end;
+        h3 {
+          margin: 0;
+          font-size: rem(20);
+          color: #555;
```

Comment:
`$color-base` or `$color-gray` (we can discuss what the default for text should be at our meeting next week, but let's use a variable)